webpack.config.js

```diff
@@ -23,12 +23,14 @@ const plugins = [
   new webpack.EnvironmentPlugin({'MOCK_STORE': null}),
   new webpack.EnvironmentPlugin(['APP']),
   new webpack.EnvironmentPlugin({'BASENAME': '/'}),
+  new webpack.EnvironmentPlugin(['LOGOUT_INACTIVE_USERS']),
   new webpack.EnvironmentPlugin(['REACT_APP_PROJECT_ID']),
   new webpack.EnvironmentPlugin(['REACT_APP_ARRANGER_API']),
   new webpack.EnvironmentPlugin(['REACT_APP_DISABLE_SOCKET']),
   new webpack.DefinePlugin({ // <-- key to reducing React's size
     'process.env': {
       'NODE_ENV': JSON.stringify(process.env.NODE_ENV || 'dev'),
+      LOGOUT_INACTIVE_USERS: !(process.env.LOGOUT_INACTIVE_USERS === 'false'),
```

Comment:
For better readability, could we change this line to `process.env. LOGOUT_INACTIVE_USERS || 'false'` (assume by default is `false`) ?

Reply:
Hmm by default it's set to true. The tricky part for me was that the bash script brings the boolean in as a string :/